<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_video_full">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>424</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string>video_full</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_video_small">
    <property name="geometry">
     <rect>
      <x>439</x>
      <y>10</y>
      <width>570</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string>video_small</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_front">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>500</y>
      <width>132</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>front</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_rear">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>500</y>
      <width>132</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>rear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_left">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>500</y>
      <width>132</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_right">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>500</y>
      <width>132</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>right</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
